src/app/action-items/action-items.component.ts

```diff
@@ -20,10 +26,14 @@ export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
   isLoadingActionItems = false;
   isLoadingActionItem = false;
-  actionItemId: string | undefined;
-  actionItem: ActionItem;
-  errorMessage: string | undefined;
-  justAddedItemId: string;
+  actionItemId?: string;
+  actionItem?: ActionItem;
+  errorMessage?: string;
+  actionItemIdForScroll: string;
```

Comment:
**SUGGESTION:** I think that if you made this variable optional then you could get rid off additional [scrollToActionItem variable](https://github.com/romatowskak/examp-ng-app/pull/4/files#diff-697b1334bd6e35d971c84803ccbe914aR33) by clearing this variable in the [line below](https://github.com/romatowskak/examp-ng-app/pull/4/files#diff-697b1334bd6e35d971c84803ccbe914aR92).
```suggestion
  actionItemIdForScroll?: string;
```